test/language/expressions/optional-chaining/member-expression-optional-chain.js

```diff
@@ -0,0 +1,51 @@
+/*---
+esid: pending
+info: |
+  optional chain on member expression
+description: >
+  Left-Hand-Side Expressions
+    OptionalExpression:
+      MemberExpression OptionalChain
+features: [optional-chaining]
+---*/
+
+const arr = [10, 11];
+const fn = (arg1, arg2) => {
+  return arg1 + arg2;
+}
+const i = 0;
+const obj = {
+  a: 'hello',
+  b: {val: 13},
+  c: (arg1) => {
+    return arg1 * 2;
+  },
```

Comment:
Same as above... non-blocking, entirely out of curiosity question: Why the arrow function here, and not just `{c(arg){ return arg * 2 }}`?

Reply:
no good reason, went ahead and used the more concise form.